app/src/androidTest/java/com/karumi/ui/view/SuperHeroDetailActivityTest.kt

```diff
@@ -34,16 +35,17 @@ class SuperHeroDetailActivityTest : AcceptanceTest<SuperHeroDetailActivity>(
     }
 
     private fun givenThereIsASuperHero(isAvenger: Boolean): SuperHero {
+        val superHeroId = "id"
```

Comment:
There is no more need to have temporal variables to give names to function arguments when we have named arguments!

```kotlin
val superHero = SuperHero(id = "id", name = "SuperHero", photo = null, isAvenger = isAvenger, description = "Super Hero Description")
```

There are a couple other places where you can use the same feature (`MainActivityTest`, `SuperHeroViewHolderTest`)

Reply:
yep I wil review and change